configs/rockpro64_emmc_config

```diff
@@ -1,8 +1,4 @@
+ROCKPRO64_CONFIG="emmc-boot"
```

Comment:
This this should be:

```
ROCKPRO64_CONFIG="rockpro64_emmc"
```

Reply:
These I haven't changed to avoid modifying the scripts for RockPro64 (as they are manually updated to S3 and not checked in in any repo).

I agree in that we should follow the same convention, so I will then modify the script and re-upload both the script tarball and the binaries.